IdentityCore/src/workplacejoin/MSIDWorkPlaceJoinUtilBase.m

```diff
@@ -38,11 +38,33 @@ + (NSString *_Nullable)getWPJStringDataForIdentifier:(nonnull NSString *)identif
                                          accessGroup:(nullable NSString *)accessGroup
                                              context:(id<MSIDRequestContext>_Nullable)context
                                                error:(NSError*__nullable*__nullable)error
+{
+    return [self getWPJStringDataFromV2ForTenantId:nil
+                                        identifier:identifier
+                                               key:nil
+                                       accessGroup:accessGroup
+                                           context:context
+                                             error:error];
+}
+
++ (NSString *_Nullable)getWPJStringDataFromV2ForTenantId:(NSString *)tenantId
+                                              identifier:(nonnull NSString *)identifier
+                                                     key:(CFStringRef)key
```

Comment:
Could you please pass NSString around everywhere in method parameters?

Reply:
We want to pass here keys to be queried on the keychain ([kSecAttrLabel](https://developer.apple.com/documentation/security/ksecattrlabel), kSecAttrService, etc) which are of type [CFString](https://developer.apple.com/documentation/corefoundation/cfstring).
If we keep argument NSString we'll have to additionally typecast them to NSString and if someone accidentally passes NSString instead of CFString, we may observe unexpected behaviour. 
Pls let me know if we should still update CFStringRef to NSString